presentation/handler/invitation_handler.go

```diff
@@ -0,0 +1,134 @@
+package handler
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	"github.com/google/uuid"
+	"split-the-bill-server/core"
+	. "split-the-bill-server/domain/service/service_inf"
+	"split-the-bill-server/presentation/dto"
+)
+
+// TODO: maybe delete invitationService if only one service is used.
+// In our handler function we could call methods from the service directly via h.MethodName.
+type InvitationHandler struct {
+	invitationService IInvitationService
+}
+
+func NewInvitationHandler(invitationService *IInvitationService) *InvitationHandler {
+	return &InvitationHandler{invitationService: *invitationService}
+}
+
+// GetByID returns the group invitation with the given ID.
+//
+//	@Summary	Get Group Invitation By ID
+//	@Tags		Invitation
+//	@Accept		json
+//	@Produce	json
+//	@Param		id	path		string	true	"Invitation ID"
+//	@Success	200	{object}	dto.GeneralResponseDTO
+//	@Router		/api/invitation/{id} [get]
+func (h InvitationHandler) GetByID(c *fiber.Ctx) error {
+	id := c.Params("id")
+	if id == "" {
+		return core.Error(c, fiber.StatusBadRequest, fmt.Sprintf(ErrMsgParameterRequired, "id"))
+	}
+	uid, err := uuid.Parse(id)
+	if err != nil {
+		return core.Error(c, fiber.StatusInternalServerError, fmt.Sprintf(ErrMsgParseUUID, id, err))
```

Comment:
I think it's more appropriate to return fiber.StatusBadRequest since the failure is related to the user providing an invalid UUID, and it's a client-side issue.

Same for "GetAllFromUser".